docs/index.rst

```diff
@@ -151,6 +151,31 @@ on a CI/build server).
 * ``TWINE_NON_INTERACTIVE`` - Do not interactively prompt for username/password
   if the required credentials are missing.
 
+Proxy Support
+^^^^^^^^^^^^^
+
+Twine allows you to upload your distributions with/without proxy via
+these environment variables:
+
+* ``HTTP_PROXY`` - Upload your distributions with HTTP proxy.
+* ``HTTPS_PROXY`` - Upload your distributions with HTTPS proxy.
+* ``NO_PROXY`` - Upload your distributions without proxy.
+
+For example:
```

Comment:
Sorry for not being clearer. I meant re-writing the whole section:

```suggestion
Twine can be configured to use a proxy by setting environment variables. For example:
```

Given the confusion around `NO_PROXY`, and that https should be used in general, I'd rather focus on the most common case for these docs.

Reply:
I see.